components/nspanel_lovelace/nspanel_lovelace.cpp

```diff
@@ -533,13 +533,183 @@ void NSPanelLovelace::render_popup_page_update_(StatefulPageItem *item) {
       }
       this->render_timer_detail_update_(item);
     });
+  } else if (item->is_type(entity_type::cover)) {
+    this->render_cover_detail_update_(item);
   } else {
     return;
   }
 
   this->send_buffered_command_();
 }
 
+// entityUpdateDetail~{entity_id}~{pos}~{pos_translation}: {pos_status}~{pos_translation}~{icon_id}~{icon_up}~{icon_stop}~{icon_down}~{icon_up_status}~{icon_stop_status}~{icon_down_status}~{textTilt}~{iconTiltLeft}~{iconTiltStop}~{iconTiltRight}~{iconTiltLeftStatus}~{iconTiltStopStatus}~{iconTiltRightStatus}~{tilt_pos}"
+void NSPanelLovelace::render_cover_detail_update_(StatefulPageItem *item) {
+  if(item == nullptr) return;
+
+  auto entity = item->get_entity();
+
+  auto cover_icons = get_icon_by_name(
+    COVER_MAP,
+    entity->get_attribute(ha_attr_type::device_class),
+    entity_cover_type::window);
+
+  auto &position_str = entity->
+    get_attribute(ha_attr_type::current_position);
+  auto &supported_features_str = entity->
+    get_attribute(ha_attr_type::supported_features);
+  auto &tilt_position_str = entity->
+    get_attribute(ha_attr_type::current_tilt_position);
+
+  uint8_t position = 0;
+  uint8_t tilt_position = 0;
+  uint8_t supported_features = 0;
```

Comment:
I think the `supported_features` value might go above 255 for covers.. maybe change this to `uint16_t` to be safe?